app/scripts/modules/core/cloudProvider/cloudProvider.registry.ts

```diff
@@ -88,6 +88,8 @@ export class CloudProviderRegistry {
 
 }
 
+export let cloudProviderRegistry: CloudProviderRegistry;
 export const CLOUD_PROVIDER_REGISTRY = 'spinnaker.core.cloudProvider.registry';
 module(CLOUD_PROVIDER_REGISTRY, [])
-  .provider('cloudProviderRegistry', CloudProviderRegistry);
+  .provider('cloudProviderRegistry', CloudProviderRegistry)
+  .run(($injector: auto.IInjectorService) => cloudProviderRegistry = <CloudProviderRegistry>$injector.get('cloudProviderRegistry'));
```

Comment:
Much shorter for sure... since the pattern in this PR is the pattern used elsewhere, I will leave it like this for now and we can have a refactor PR to address all the cases of this (maybe in the same PR as a remove-all-$inject?!)